### ISSUE TITLE
Regarding cross silo scenarios and privacy preserving technologies

### ISSUE THREAD

**Issue by chanfm81** [adap/flower#706]
Hi,

Thanks for making the flower framework available for us to try.

I have some queries on usage of flower as below:
1. As flower is meant for cross device training under heterogeneous settings, can flower be used in the cross silo scenarios where the clients are organizations and the number of clients are fixed with no dropout?
2. The default data partitioning is horizontal FL. In order to achieve vertical FL, how easy is for flower to adopt such use case?
3. If aggregation is to take place in a secured manner such as using MPC, TEE and HE, are there any abstraction classes for users to customize their own security algorithms?
4. Are there any mechanisms for clients to get authenticated first before connecting to the central server?
5. Are there any examples which use xgboost or logistic regression in a federated manner using flower?

Thanks.
